test_scripts/Resumption/InteriorVehicleData/004_InteriorVD_resumption_all_modules_without_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,56 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect
+--  in case app was subscribed to all default modules (without moduleId)
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to all default modules via GetInteriorVehicleData(moduleType)
+--
+-- Sequence:
+-- 1. Transport disconnect and reconnect are performed
+-- 2. App starts registration with actual hashId after unexpected disconnect
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, moduleType, default_moduleId) to HMI during resumption data
+--    for each default module
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
please add SDL get all successful responses for request with isSubscribed=true before SDL respond RAI(success=true, result code = SUCCESS) to mobile

Reply:
Done